pages/en/lb4/Booting-an-Application.md

```diff
@@ -0,0 +1,178 @@
+---
+lang: en
+title: 'Booting an Application'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Booting-an-Application.html
+summary:
+---
+
+## What does it mean to boot an Application?
+
+A typical LoopBack application is made up of many artifacts in different files,
+organized in different folders. **Booting an Application** means:
+
+* Discover artifacts automatically based on a convention (a specific folder
+  containing files with a given suffix)
+* Process those artifacts automatically (this usually means binding them to the Application)
+
+`@loopback/boot` provides a Bootstrapper that uses Booters to automatically
+discover and bind artifacts, all packaged in an easy to use Mixin.
+
+### What is an artifact?
+
+An artifact is any LoopBack construct usually defined in code as a Class. The word
+artifact describes Controllers, Repositories, Models, etc.
+
+## Usage: @loopback/cli
+
+New projects generated using `@loopback/cli` or `lb4` are automatically enabled
```

Comment:
I wonder if it's a good idea to let the readers know up front that for general use, artifacts are automatically loaded for you and you don't need to know too much details.